src/smokeTest/java/uk/gov/hmcts/futurehearings/hmi/smoke/updateuser/UpdateUserApiSmokeTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.futurehearings.hmi.smoke.updateuser;
```

Comment:
refactor to User rather than updateUser as we should name based on the resource rather than operation nd Resource wish is a SOAP Style of coding....